app/controllers/home_controller.rb

```diff
@@ -0,0 +1,25 @@
+class HomeController < ApplicationController
+
+  def top
+    @hands = Cards.new(nil)
+  end
+  def poker
```

Comment:
controller's task is calling some methods. please define name to help understanding what this method do